shared-bindings/audiobusio/PDMIn.c

```diff
@@ -49,7 +49,7 @@
 //|
 //|   :param ~microcontroller.Pin clock_pin: The pin to output the clock to
 //|   :param ~microcontroller.Pin data_pin: The pin to read the data from
-//|   :param int frequency: Target frequency of the resulting samples. Check `frequency` for real value.
+//|   :param int frequency: Target frequency of the resulting samples. Check `frequency` for real value. Must be minimum of 16000
```

Comment:
Instead of adding a note here, how about a full sentence before the first `:param:`? That way it can explain the total 1mhz frequency requirement that is determined by `frequency * oversample`.